### ISSUE TITLE
[Docs OR IDE] `dbg_sprite_button` label argument

### ISSUE THREAD

**Issue by tinkerer-red** [YoYoGames/GameMaker-Bugs#8059]
### Description

to make use of `dbg_sprite_button` you must provide a label argument, how ever this argument is unused in the actual result. This causes conflicts with what the manual reports as the examples clearly state the first argument is a handle to a function, and no argument for the label is listed or explained.

Example of label being unused, hovering over it also provides no pop up name:
![image](https://github.com/user-attachments/assets/4e601dc4-0e15-41ef-9dbe-e73fc16741dd)

### Expected Change

Likely this is an IDE bug, but if the intent was to expand this to have hover information for sprite buttons then it would make sense to update the docs and implement that feature.

### Steps To Reproduce

descrepancy can be seen [here](https://manual.gamemaker.io/monthly/en/GameMaker_Language/GML_Reference/Debugging/dbg_sprite_button.htm) and when comparing the ide's results.
![image](https://github.com/user-attachments/assets/e754d85c-6035-4c79-823e-e9e426369ab2)
here is the same example code from the manual with the notes of with and with out the label.
```gml
sprite = sprite_index;
image = image_index;

var ref_to_sprite = ref_create(self, "sprite");
var ref_to_image = ref_create(self, "image");

toggle = function()
{
    image = (image == 0) ? 1 : 0;
};

//works, but never uses the label
dbg_sprite_button("sprite button", toggle, ref_to_sprite, ref_to_image);

//doesnt work
dbg_sprite_button(toggle, ref_to_sprite, ref_to_image);
```

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.8.1 (Monthly)

### Which operating system(s) are you seeing the problem on?

Windows 10

### Are you running GameMaker from inside your Steam library?

None

### Sample Package Attached?

- [X] I have attached my bug package

### Sample Project Added?

- [X] I have included a small sample project

**Comment by rwkay** [comment]
Fixed in 2024.11 - I have removed the `label` argument from the runner to match the documentation.

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.1100.0.654  Runtime v2024.1100.0.680
